local_init.bash

```diff
@@ -1,14 +1,19 @@
 #!/bin/bash
+
+set -xe
+
 CONTAINER_ID=$(docker container ls | grep enigmampc/client | cut -d' ' -f1)
-mkdir -p ~/.enigma
-mkdir -p ../build
-docker cp $CONTAINER_ID:/root/build/contracts ../build
-sed -e 's_http://contract_http://localhost_g;s_http://bootstrap_http://localhost_g' ../build/contracts/addresses.json > ../build/contracts/addresses.new
-mv ../build/contracts/addresses.new ../build/contracts/addresses.json
+
+rm -rf ../build ~/.enigma /tmp/enigma
```

Comment:
Please do not remove `~/.enigma`. This folder is outside this project, and may contain other files from other enigma-related projects that store information there. As you can see in the previous iteration of this code, it only created the folder if it did not exist, but otherwise leaves the contents of that folder intact. If your code does not use it, then do not delete it.